src/components/DisplayEpisodes/DisplayEpisodes.spec.js

```diff
@@ -8,25 +8,24 @@ import {nanoid} from 'nanoid';
 import DisplayEpisodes from './';
 
 describe('DisplayEpisodes component', () => {
-	it('should display the episodes name', () => {
-		const episodes = [
-			{
-				title: 'CSS Proposals @when, CSS Masonry, Carets',
-				url: 'https://traffic.libsyn.com/syntax/Syntax_-_497.mp3',
-				date: '15.08.2022',
-				length: '00:20:11',
-				key: nanoid(),
-			},
-		];
-		render(<DisplayEpisodes episodes={episodes} />);
+	it('should display the episodes title, date and length', () => {
+		const episodes = {
+			title: 'CSS Proposals @when, CSS Masonry, Carets',
+			url: 'https://traffic.libsyn.com/syntax/Syntax_-_497.mp3',
+			date: '15.08.2022',
+			length: '00:20:11',
+			key: nanoid(),
+		};
 
-		episodes.forEach(episode => {
-			const episodeTitle = screen.getByText(episode.title);
-			expect(episodeTitle).toBeInTheDocument();
-			const episodeDate = screen.getByText(episode.date);
-			expect(episodeDate).toBeInTheDocument();
-			const episodeLength = screen.getByText(episode.length);
-			expect(episodeLength).toBeInTheDocument();
-		});
+		// The component expects an array as prop.
+		const episodesProp = [episodes];
+		render(<DisplayEpisodes episodes={episodesProp} />);
```

Comment:
See my comment above 🙂 
```suggestion
		render(<DisplayEpisodes episodes={episodes} />);
```